benchto-driver/src/main/java/io/trino/benchto/driver/BenchmarkProperties.java

```diff
@@ -233,4 +241,9 @@ else if (booleanString.equalsIgnoreCase(Boolean.FALSE.toString())) {
             throw new IllegalStateException(String.format("Incorrect boolean value: %s.", booleanString));
         }
     }
+
+    public enum RepeatLevel {
+        QUERY,
```

Comment:
I think it should most likely be:
* BENCHMARK (repeat queries within benchmark run)
* SUITE (repeat queries withing entire suite execution)

Reply:
good point.